ACPs/77-reinventing-subnets/README.md

```diff
@@ -42,134 +44,133 @@ For messages produced by the P-Chain for a given Subnet, only that Subnet's vali
+|     `validators` | `[]ValidatorData` |                          4 + sum(`validatorLengths`) bytes |
+|                  |                   | 74 + len(`managerAddress`) + len(`validatorLengths`) bytes |
 
 - `codecID` is the codec version used to serialize the payload, and is hardcoded to `0x0000`
 - `sum(validatorLengths)` is the sum of the lengths of `ValidatorData` serializations included in `validators`.
 - `subnetID` identifies the Subnet that is being converted (described further below).
 - `managerChainID` and `managerAddress` identify the validator manager for the given Subnet. This is the (blockchain ID, adress) tuple allowed to send Warp messages to modify the Subnet's validator set.
 - `validators` are the initial pay-as-you-go validators for the given Subnet.
 
-The `SubnetConversionMessage` is specified as an `AddressedCall` with `sourceChainID` set to the P-Chain ID, the `sourceAddress` set to an empty byte array, and a payload of:
+The `L1ConversionMessage` is specified as an `AddressedCall` with `sourceChainID` set to the P-Chain ID, the `sourceAddress` set to an empty byte array, and a payload of:
 
-| Field | Type | Size |
-| -: | -: | -: |
-| `codecID` | `uint16` | 2 bytes |
-| `typeID` | `uint32` | 4 bytes |
+|                Field |       Type |     Size |
+| -------------------: | ---------: | -------: |
+|            `codecID` |   `uint16` |  2 bytes |
+|             `typeID` |   `uint32` |  4 bytes |
 | `subnetConversionID` | `[32]byte` | 32 bytes |
-| | | 38 bytes |
+|                      |            | 38 bytes |
 
 - `codecID` is the codec version used to serialize the payload, and is hardcoded to `0x0000`
 - `typeID` is the payload type identifier and is `0x00000000` for this message
-- `subnetConversionID` is the SHA256 hash of the `SubnetConversionData` from a given `ConvertSubnetTx`
+- `subnetConversionID` is the SHA256 hash of the `SubnetConversionData` from a given `ConvertSubnetToL1Tx`
 
-#### `RegisterSubnetValidatorMessage`
+#### `RegisterL1ValidatorMessage`
 
-The P-Chain can consume a `RegisterSubnetValidatorMessage` from validator managers through a `RegisterSubnetValidatorTx` to register an addition to the Subnet's validator set.
+The P-Chain can consume a `RegisterL1ValidatorMessage` from validator managers through a `RegisterL1ValidatorTx` to register an addition to the Subnet's validator set.
 
 The following is the serialization of a `PChainOwner`:
 
-| Field | Type | Size |
-| -: | -: | -: |
-| `threshold` | `uint32` | 4 bytes |
-| `addresses` | `[][20]byte` | 4 + len(`addresses`) * 20 bytes |
-| | | 8 + len(`addresses`) * 20 bytes |
+|       Field |         Type |                             Size |
+| ----------: | -----------: | -------------------------------: |
+| `threshold` |     `uint32` |                          4 bytes |
+| `addresses` | `[][20]byte` | 4 + len(`addresses`) \* 20 bytes |
+|             |              | 8 + len(`addresses`) \* 20 bytes |
```

Comment:
(no change expected): Why was `*` changed to `\*`? Is this a best-practice for markdown to avoid bolding text by accident?